### ISSUE TITLE
Scraped CSV pandas dataframe I get: ValueError('Length of values does not match length of ' 'index')

### ISSUE THREAD

**Issue by jytinjalmon** [pandas-dev/pandas#18655]
Does Pandas not support scraped csv's, but only manual entry?  I've written a SO question about it.  I had posted all content here, but then page refreshed itself.

https://stackoverflow.com/questions/47666070/scraped-csv-pandas-dataframe-i-get-valueerrorlength-of-values-does-not-match

**Comment by gfyoung** [comment]
@jytinjalmon : Thanks for reporting this!  Unfortuantely, I'm not sure I really understand your question, since scraped or not, we will read so long as the CSV is valid.

Secondly, I tried reading your SO post, but the code is very hard to follow since you have it somewhat all over the place.  If you could try to post a clean version of your code and attach the CSV's in this post (just upload via GitHub as .txt files), that would be great.

**Comment by TomAugspurger** [comment]
I think this was answered on SO.
